contrib/example.tengo

```diff
@@ -1,2 +1,13 @@
+/*
+variables available 
+read-only:
+inAccount, inProtocol, inChannel, inGateway
+outAccount, outProtocol, outChannel, outGateway
+
+read-write:
+msgText, msgUsername
+*/
+
```

Comment:
The example.tengo is for the `InMessage` which has other variables: 
https://github.com/42wim/matterbridge/blob/1e8b537d613edf18b6b73834796967eacaa1e03c/matterbridge.toml.sample#L1473-L1491